src/components/side_panel/custom_currency/custom_currency.xml

```diff
@@ -59,11 +59,11 @@
           onChange.bind="toggleAccountingFormat"
         />
         <div class="o-format-examples mt-4" t-if="selectedFormat">
-          <table>
+          <table class="w-100">
```

Comment:
I think that it can sync nicely with https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout#fixed without having the weird max-width = 1px

Reply:
good catch :+1: